experimental/auth-logic/provenance_wrapper.go

```diff
@@ -28,29 +28,26 @@ import (
 
 type provenanceWrapper struct{ filePath string }
 
-func (p provenanceWrapper) EmitStatement() UnattributedStatement {
-	// TODO(#40) return the errors produced here instead once the
-	// wrapper interface has been fixed.
-	provenance, provenanceErr := slsa.ParseProvenanceFile(p.filePath)
-	if provenanceErr != nil {
-		panic(provenanceErr)
+func (p provenanceWrapper) EmitStatement() (UnattributedStatement, error) {
+	provenance, err := slsa.ParseProvenanceFile(p.filePath)
+	if err != nil {
+		return UnattributedStatement{}, err
 	}
 
 	if len(provenance.Subject) < 1 {
 		noSubjectError := errors.New("Provenance file missing subject")
```

Comment:
```suggestion
		noSubjectError := fmt.Error("Provenance file missing subject")
```

In the rest of the code-base we use `fmt.Errorf`, and I think that is the more conventional way of creating error messages. This applies file-wide.

Reply:
fixed